### DISCUSSION TITLE
Apple silicon GPU acceleration

### DISCUSSION THREAD

**Discussion by mcanderson179** [utmapp/UTM#5466] [Development]
I'm new to virtualization. Will we get a GPU acceleration fix anytime soon? Not for gaming, I just want to use a cool terminal emulator with the most responsiveness I can.

**Comment by tifasoftware** [comment]
The GPU Accerlation should work for Linux, choose the cards with -gl. We just fixed a major issue with it nearly 4 months ago, unless it broke again.

**Comment by mcanderson179** [comment]
Ah I see. I realize I just needed to update. Thank you!
